build.gradle

```diff
@@ -16,6 +16,10 @@ plugins {
 
 repositories {
     mavenCentral()
+
+    maven {
+        url "https://artifactory.broadinstitute.org/artifactory/libs-snapshot/" // for intel deflater snapshots
```

Comment:
I would vote for excluding it from the dependency:
https://docs.gradle.org/current/userguide/dependency_management.html#sub:exclude_transitive_dependencies